ocp_resources/limit_range.py

```diff
@@ -0,0 +1,27 @@
+# API reference: https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-limit-ranges.html
```

Comment:
this is a k8s resource, please change to the relevant api doc (https://kubernetes.io/docs/reference/kubernetes-api/policy-resources/limit-range-v1/)

Reply:
ack